PSModule/Helper.cs

```diff
@@ -40,14 +40,12 @@ class Helper
         private const string NO_OF_TESTS = "No. of tests";
         private const string PASSING_RATE = "Percentage of tests";
         private const string STYLE = "style";
-        private const string UFT_REPORT = "UFT report";
+        private const string UFT_REPORT_ = "UFT report";

```

Comment:
It looks like this field can be removed and the new one `UFT_REPORT` can be used instead. The only difference is the case sensitive of word **report**.